api-scripts/construct_mappreview.py

```diff
@@ -0,0 +1,367 @@
+import json
+import logging
+import os
+import urllib.error
+import urllib.request
+import requests
+import warnings
+import yaml
+
+import pygeoprocessing  # mamba install pygeoprocessing
+from osgeo import ogr
+from osgeo import osr
+from rio_tiler.io import Reader # mamba install rio-tiler
+
+
+logging.basicConfig(level=logging.DEBUG)
+LOGGER = logging.getLogger(os.path.basename(__file__))
+
+
+def get_map_settings(layers: list[dict]) -> dict:
+    """Get map default minzoom, maxzoom, and bounds based on layers to display"""
+    minzoom = 1
+    try:
+        minzoom = min(filter(None, [l.get('minzoom') for l in layers]))
+    except Exception as e:
+        LOGGER.debug(f"exception with minzoom: {e}")
```

Comment:
`DEBUG` seems like a pretty low level for this ... maybe `WARNING` would help it stand out in the logs and emphasize the severity?